pegen/c_generator.py

```diff
@@ -236,7 +241,9 @@ def generate(self, filename: str) -> None:
                 self.print()
                 self.visit(rule)
         mode = int(self.rules["start"].type == "mod_ty")
-        self.print(EXTENSION_SUFFIX.rstrip("\n") % dict(mode=mode))
```

Comment:
ISTM that this encourages people to copy the entire body of `EXTENSION_SUFFIX` in order to change the module name. Perhaps we can add a variable to override just that, so fewer people will have to set `@trailer`?

Reply:
Good idea!